test/integration/test/philips_light_test.js

```diff
@@ -29,7 +30,7 @@ describe('Initiate the connection with foxbox as Philips Hue hub',function(){
     philipshue_server.turnOffLight(2);
     philipshue_server.turnOffLight(3);
 
-    // kick off the foxbox within 10 seconds
+    // kick off the foxbox within 3 seconds
```

Comment:
I don't see this value coded somewhere. Is is an approximation of the time take, or is it a hard deadline?
I'd clarify the comment.